Antidote/AbstractCallViewController.m

```diff
@@ -106,6 +107,7 @@ - (void)installConstraints
         make.top.equalTo(self.topViewContainer.topMargin).with.offset(kIndent);
         make.centerX.equalTo(self.topViewContainer);
         make.height.equalTo(kNameLabelHeight);
+        make.width.lessThanOrEqualTo(self.view.bounds.size.width / 2.0);
```

Comment:
`nameLabel` width will be assigned to constant and won't be updated on `view` frame change (in landscape?). I think it would be better to
- remove `centerX` and `width`
- set `left` to `view.left` with some offset
- set `right` to `view.right` with some offset
- set `textAlignment` property of UILabel to `NSTextAlignmentCenter`